src/pymor/models/basic.py

```diff
@@ -73,8 +73,10 @@ class StationaryModel(ModelBase):
     rhs
         The vector F. Either a |VectorArray| of length 1 or a vector-like
         |Operator|.
-    outputs
-        A dict of additional output |Functionals| associated with the model.
+    output_functional
+        |Operator| mapping a given solution to the model output. In many applications,
+        this will be a |Functional|, i.e. an |Operator| mapping to scalars.
+        This is not required, however.
```

Comment:
Why restrict to a single output functional?

Reply:
Note that you can have multiple output functionals by using a vector-valued `Operator`, similar to the `C` operator in `LTIModel`. IMHO the whole point of this PR is to have a well-defined and not to complicated notion of output in pyMOR, and this is exactly the sacrifice we are willing to make to achieve this. If `StationaryModel` (or `InstationaryModel`) had multiple output functionals, it would not be clear what `output` should return or what `output_space` shall be.

Reply:
Ah, I was confused because of the "functional" in the name... Actually, I am for having a vector-valued output.

Reply:
Yeah `output_operator` would be more correct, but the RB guys always speak of output functionals (and often there is only one), hence this compromise.